### PR TITLE
Move fuzzer building to upstream.

### PR DESCRIPTION
**Pull request by vrabaud**

### CODE REVIEW

**Review by maryla-uc — APPROVED** [review]

**Comment by vrabaud** [comment]
@wantehchang , please review, thx !

**Review by jonathanmetzman — APPROVED** [review]
LGTM

File: projects/libavif/build.sh [hunk 1]
```diff
@@ -15,22 +15,7 @@
...
-ninja
-
-# build fuzzer
-$CXX $CXXFLAGS -std=c++11 -I../include \
-    ../tests/oss-fuzz/avif_decode_fuzzer.cc -o $OUT/avif_decode_fuzzer \
-    $LIB_FUZZING_ENGINE libavif.a ../ext/dav1d/build/src/libdav1d.a
-
-# copy seed corpus
-cp $SRC/avif_decode_seed_corpus.zip $OUT/
+bash tests/oss-fuzz/build.sh
```
Comment (wantehchang) [inline projects/libavif/build.sh:18]: Should we also move lines 20-21 into tests/oss-fuzz/build.sh?

Unless tests/oss-fuzz/build.sh is used elsewhere, we don't need to split the original contents of this file into two parts. It seems cleaner to have this file delegate everything to tests/oss-fuzz/build.sh.

**Review by wantehchang — COMMENTED** [review]
LGTM.

File: projects/libavif/build.sh (hunk 1 above)
Comment (vrabaud) [reply projects/libavif/build.sh:18]: I thought about it but I wanted to have the build.sh file be about building as much as possible, and as agnostic to oss-fuzz as possible. Hence adding a section with $OUT would have hurt.
We can move it later, it is non-critical. Or have more logs here :)

**Pull request merged**